### PR TITLE
QUICKSTART updates

### PR DESCRIPTION
**Pull request by bklaas**
Updated the QUICKSTART for the admin side. Deferred some initial setup steps to the notifications-api side since that's required for admin anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,28 +9,41 @@ US Notify admin application - https://notifications-admin.app.cloud.gov (contact
...
+Open the notifications-admin repo in VS Code (File->Open Folder, select notifications-admin folder)
 
-# make sure your docker daemon is running
+create a .env file using sample.env as a template
+`cp sample.env .env` (or via VS Code file browser)
 
-# Using the command pallette (cmd+shift+p), search "Remote Containers: Open folder in project" 
-# choose devcontainer-admin folder, after reload, hit "show logs" in bottom-right
-# logs should complete shortly after running gulp.js and compiling front-end files
+from the notifications-api checkout, copy the values in that repo's .env file for `AWS_ACCESS_KEY_ID1` and `AWS_SECRET_ACCESS_KEY` into this repo's .env file.
```
Comment (rahearn) [inline README.md:22]:
```suggestion
from the notifications-api checkout, copy the values in that repo's .env file for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` into this repo's .env file.
```

File: README.md [hunk 2]
```diff
@@ -9,28 +9,41 @@ US Notify admin application - https://notifications-admin.app.cloud.gov (contact
...
 
+---
+## .env Setup
+
+create a .env file using sample.env as a template
+`cp sample.env .env` (or via VS Code file browser)
+
+from the notifications-api checkout, copy the values in that repo's .env file for `AWS_ACCESS_KEY_ID1` and `AWS_SECRET_ACCESS_KEY` into this repo's .env file.
+
+Change `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD` to what you'd like them to be for this deployment.
```
Comment (rahearn) [inline README.md:45]: This `.env Setup` section seems duplicated from above

**Review by rahearn — CHANGES_REQUESTED** [review]
Looks good other than the duplicated env setup stuff

File: README.md (hunk 2 above)
Comment (bklaas) [reply README.md:40]: Leaving this section as standalone and adding a pointer in the quickstart section to it.

**Review by jimmoffet — APPROVED** [review]

**Review by rahearn — APPROVED** [review]

**Pull request merged**